docs/src/components/domains.md

```diff
@@ -0,0 +1,30 @@
+# Domains
+
+A **Valence Program** is an instance of the **Valence Protocol**. A Valence program's execution can typically _span multiple Blockchains_. In the Valence Protocol, we refer to the various Blockchains that are supported as **domains**.
```

Comment:
Not sure if the term "Valence Protocol" is going to be used. Probably worth asking Udit

Reply:
Fair enough .. I will re-phrase it. Valence accounts actually use the term `approve a library`.